clients/src/main/java/org/apache/kafka/common/telemetry/internals/ClientTelemetryReporter.java

```diff
@@ -362,7 +362,7 @@ public long timeToNextUpdate(long requestTimeoutMs) {
                     throw new IllegalStateException("Unknown telemetry state: " + localState);
             }
 
-            log.debug("For telemetry state {}, returning the value {} ms; {}", localState, timeMs, msg);
+            log.trace("For telemetry state {}, returning the value {} ms; {}", localState, timeMs, msg);
```

Comment:
Do you think it makes sense to rephrase the log? For the `msg` can we explicitly say:

...client will wait for {}ms before submitting the next...

Reply:
so that we can remove the "returning the value 224678 ms" - it was a bit hard to understand what it actually mean.

Reply:
I don't think it will make much difference, seems fine to me. Writing the suggestion will involve changing `msg` in each block. I think it's fine the way it is :)